theme/parts/toolbox.css

```diff
@@ -61,13 +61,16 @@ findbar:-moz-window-inactive description,
 
 /* Reorder toolbars */
 #navigator-toolbox #nav-bar {
-	-moz-box-ordinal-group: 0;
+	-moz-box-ordinal-group: 98;
+	order: 98;
```

Comment:
Is the `order` property needed? I mean, we only target one Firefox version at a time, so we should only keep one property, whatever works.

Reply:
It didn't work if `order` was not including. I was constantly messing around with it to work and that's the only way it had the intended result.

Reply:
And if you remove `-moz-box-ordinal-group` it also doesn't work?

Reply:
Without `-moz-box-ordinal-group`, `tabsAsHeaderbar` still doesn't work. But the tabs are located beneath the urlbar.

Reply:
I mean if `order` works and `-moz-box-ordinal-group` doesn't, then let's remove  `-moz-box-ordinal-group`  everywhere.

Reply:
`tabsAsHeaderbar` could be worked later, I can ask the contributor of that feature to take care of it.

Btw have you tried using `!important`?

Reply:
> Btw have you tried using !important?

With `-moz-box-ordinal-group`? It did not seem to work, no. I am not too knowledgeable with Firefox development, maybe this is a recent change. I have no idea.

Reply:
> `tabsAsHeaderbar` could be worked later, I can ask the contributor of that feature to take care of it.

Ok, sounds good!

Reply:
> > Btw have you tried using !important?
> 
> With `-moz-box-ordinal-group`? It did not seem to work, no. I am not too knowledgeable with Firefox development, maybe this is a recent change. I have no idea.

Yeah, they probably removed their custom prop. Looks good then.